scripts/skype

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-PATH=/usr/sbin:/usr/bin:/sbin:/bin
+PATH=/usr/sbin:/usr/bin:/sbin:/bin:/usr/local/sbin:/usr/local/bin
```

Comment:
@darashenka this change puts the skype script in an infinite loop. It is not required and should be removed.